### PR TITLE
feat(iam): adds provider to check if user has 5 or more GitHub repos

### PR DESCRIPTION
**Pull request by aminah-io**

### CODE REVIEW

File: iam/src/providers/fiveOrMoreGithubRepos.ts
Comment (gdixon) [inline iam/src/providers/fiveOrMoreGithubRepos.ts:50]: It might be better to have this as a statement that matches the range rather than the number of repos that the user has - if the user creates a new repo, they would get a new hash, and would be able to use that new stamp against a different Passport.

**Review by gdixon — CHANGES_REQUESTED** [review]
Just the one tiny comment - the rest looks awesome! Nice work @aminah-io 🚀

**Review by gdixon — APPROVED** [review]
LGTM!! Nicely done 🚀

**Review by farque65 — APPROVED** [review]
LGTM once the merge conflicts are resolved

**Pull request merged**
